package.json

```diff
@@ -12,7 +12,8 @@
     "license": "lerna exec -- cp ../../LICENSE .",
     "release": "yarn license && lerna publish from-package",
     "serve": "serve packages/vega/ -l 8080",
-    "test": "lerna run test && yarn lint",
+    "test": "yarn run test && yarn run lint",
```

Comment:
This looks circular: `yarn test` calls `yarn run test`? Should this be `yarn test:no-lint && yarn lint`?

Reply:
Yeah, you are right, thanks.